docs/docs/icechunk-python/parallel.md

```diff
@@ -16,38 +16,38 @@ including those executed remotely in a multi-processing or any other remote exec
 Here is how you can execute such writes with Icechunk, illustrate with a `ThreadPoolExecutor`.
 First read some example data, and create an Icechunk Repository.
 
-```python
+```python exec="on" session="parallel" source="material-block"
 import xarray as xr
 import tempfile
 from icechunk import Repository, local_filesystem_storage
 
 ds = xr.tutorial.open_dataset("rasm").isel(time=slice(24))
-repo = Repository.create(local_filesystem_storage(tempfile.mkdtemp()))
+repo = Repository.create(local_filesystem_storage(tempfile.TemporaryDirectory().name))
 session = repo.writable_session("main")
 ```
 
 We will orchestrate so that each task writes one timestep.
 This is an arbitrary choice but determines what we set for the Zarr chunk size.
 
-```python
+```python exec="on" session="parallel" source="material-block" result="code"
 chunks = {1 if dim == "time" else ds.sizes[dim] for dim in ds.Tair.dims}
```

Comment:
can we turn off output for this one?
<img width="610" alt="image" src="https://github.com/user-attachments/assets/fdacc6f8-0439-4a68-9268-fc8655ae076e" />

there are a couple more places we'll need to do that on this page.